json_2_yolo.py

```diff
@@ -0,0 +1,34 @@
+# This script can convert your AU-AIR annotations to YOLO Format
+import json
+data = open(r'annotations.json') #Pass in the path for AU-AIR annotation file
+ann_file = json.load(data)
+ann_list = ann_file['annotations']
+b = [i.get('bbox') for i in ann_list] #b is a list which contains the bbox parameters for YOLO Conversion
+
+# print(b)
+
+name = []
+for i in ann_file['annotations']:
+    name.append(i.get('image_name'))
+
+# print(name)
+YOLO_LABELS_PATH = r'yolo_annotation_path'  # Pass in a path to save the YOLO Annotation Files
```

Comment:
The output of this line is a string ("yolo_annotation_path"). Instead give a destination folder.

Reply:
Hi, actually for my case I had created a folder named "yolo_annotation_path", but yeah it might have been confusing.

Reply:
Please feel free to review the new code